WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -732,6 +734,11 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         } else if (preference == mTimezonePref) {
             setTimezonePref(newValue.toString());
             mSiteSettings.setTimezone(newValue.toString());
+        } else if (preference == mGutenbergDefaultForNewPosts) {
+            mDispatcher.dispatch(SiteActionBuilder.newDesignateMobileEditorAction(
+                    new DesignateMobileEditorPayload(mSite, ((Boolean) newValue) ? "gutenberg" : "aztec")));
```

Comment:
WDYT about declaring the editor payload literals ("gutenberg" and "aztec") somewhere as string constants? It seems they are used in a few different places throughout the codebase.

Reply:
Fixed with 77f01f06578772c079abee27e94a8076292d338c.